lib/src/visitor/async_evaluate.dart

```diff
@@ -282,6 +282,32 @@ class _EvaluateVisitor
         return SassBoolean(_environment.content != null);
       }),
 
+      BuiltInCallable("module-variables", r"$module", (arguments) {
+        var namespace = arguments[0].assertString("module");
+        var module = _environment.modules[namespace.text];
+        if (module == null) {
+          throw 'There is no module with namespace "${namespace.text}".';
+        }
+
+        return SassMap({
+          for (var entry in module.variables.entries)
+            SassString(entry.key): entry.value
```

Comment:
Any danger of exposing something you don't want to by returning this `Value` through the map without reboxing it in some way?

Reply:
No, `Value`s are intentionally only ever normal Sass values.